exampleSite/config.toml

```diff
@@ -37,7 +37,10 @@ paginate = 10
 
     about_us = "<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>"
     copyright = "Copyright (c) 2015 - 2016, YourCompany; all rights reserved."
-    date_format = "2006-01-02"
```

Comment:
Hi @adrianmo, maybe in the views we need to prevent if the user forget the parameter in the configuration file or the format is wrong?